Value too long for column "SUCCEEDED_NODE_LIST VARCHAR(255)

# Bug report

Upgraded to 2.7.1 from 2.6.11, jobs not completing

**My Rundeck detail**

* Rundeck version: 2.7.1
* install type: rpm (rpm,deb,war,launcher)
* OS Name/version: CentOS 7

**Expected Behavior**

Jobs finishing successfully. They do complete all the steps but rundeck stays in a running job state

**Actual Behavior**

Job stays running and the log shows:

Value too long for column "SUCCEEDED_NODE_LIST VARCHAR(255) ... All node names ... (14155)

rundeck-config.properties:

dataSource.url = jdbc:h2:file:/var/lib/rundeck/data/rundeckdb;MVCC=true
quartz.props.threadPool.threadCount = 30

I am having the same issue.  I found that if I run a job agains a few nodes, it complete successfully.  If the job runs agains 100+ nodes I get this:

2016-12-08 07:54:01,467 [qtp1480010240-17] INFO  grails.app.services.rundeck.services.ScheduledExecutionService - scheduling temp job: TEMP:lmagstad:42582
2016-12-08 07:55:12,101 [quartzScheduler_Worker-4] INFO  grails.app.services.rundeck.services.ExecutionUtilService - Execution successful: 42582 in project XXX
2016-12-08 07:55:12,111 [quartzScheduler_Worker-4] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper - ERROR: value too long for type character varying(255)
2016-12-08 07:55:12,114 [quartzScheduler_Worker-4] ERROR grails.app.jobs.rundeck.quartzjobs.ExecutionJob - Execution 42582 save result status: caught exception: Hibernate operation: could not execute statement; SQL [n/a]; ERROR: value too long for type character varying(255); nested exception is org.postgresql.util.PSQLException: ERROR: value too long for type character varying(255)
org.springframework.dao.DataIntegrityViolationException: Hibernate operation: could not execute statement; SQL [n/a]; ERROR: value too long for type character varying(255); nested exception is org.postgresql.util.PSQLException: ERROR: value too long for type character varying(255)
        at rundeck.services.ReportService.reportExecutionResult(ReportService.groovy:65)
        at rundeck.services.ExecutionService.logExecution(ExecutionService.groovy:843)
        at rundeck.services.ExecutionService$_saveExecutionState_closure61.doCall(ExecutionService.groovy:2227)
        at rundeck.services.ExecutionService.saveExecutionState(ExecutionService.groovy:2148)
        at rundeck.quartzjobs.ExecutionJob$_saveState_closure4.doCall(ExecutionJob.groovy:509)
        at rundeck.quartzjobs.ExecutionJob.withRetry(ExecutionJob.groovy:447)
        at rundeck.quartzjobs.ExecutionJob.saveState(ExecutionJob.groovy:519)
        at rundeck.quartzjobs.ExecutionJob.execute_internal(ExecutionJob.groovy:162)
        at rundeck.quartzjobs.ExecutionJob$_execute_closure1.doCall(ExecutionJob.groovy:90)
        at com.codahale.metrics.Timer.time(Timer.java:99)
        at rundeck.quartzjobs.ExecutionJob.execute(ExecutionJob.groovy:89)
        at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
        at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: org.postgresql.util.PSQLException: ERROR: value too long for type character varying(255)
        at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2157)
        at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1886)
        at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
        at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:559)
        at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:417)
        at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:363)
        ... 13 more

--- a/rundeckapp/grails-app/domain/rundeck/ExecReport.groovy
+++ b/rundeckapp/grails-app/domain/rundeck/ExecReport.groovy
@@ -32,6 +32,9 @@ class ExecReport extends BaseReport{
 
     static mapping = {
         adhocScript type: 'text'
+        filterApplied type: 'text'
+        succeededNodeList type: 'text'
+        failedNodeList type: 'text'
     }
 
     static constraints = {

